### PR TITLE
feat: finish kitex2kitex crosstest

### PR DESCRIPTION
**Pull request by DMwangnima**
#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.
- [ ] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io)

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

File: pkg/codec.go [hunk 1]
```diff
@@ -443,3 +443,23 @@ func readBody(header *dubbo_spec.DubboHeader, in remote.ByteBuffer) ([]byte, err
 	}
 	return in.Next(length)
 }
+
+func getJavaClassName(message remote.Message) string {
```
Comment (felix021) [inline pkg/codec.go:455]: Better provide a more comprehensive warning to instruct the user to add the annotation in his IDL.

Comment (DMwangnima) [reply pkg/codec.go:455]: Ack.

File: pkg/codec.go [hunk 2]
```diff
@@ -31,6 +31,14 @@ import (
 	"github.com/kitex-contrib/codec-dubbo/pkg/iface"
 )
 
+var annotationPrompt = `
+Please add JavaClassName annotation as Dubbo Interface Name for %s service.
+Assumed Interface Name is org.apache.dubbo.api.UserProvider, api.thrift would be:
```
Comment (felix021) [inline pkg/codec.go]: `Assuming` ... `should` be

Comment (DMwangnima) [reply pkg/codec.go]: Ack.

**Review by felix021 — APPROVED** [review]

**Pull request merged**